jauco/jester
# webpackAlertFilters

## Prerequisites
- [ ] This build on PR #23, so review/merge that one first.
## Description

Adding support to suppress webpack soft errors and CriticalDependenciesWarnings.

**Note:** this affects anyone who uses `webpackWarningFilters` in their `jester.json`.
Their existing filter config will do nothing until they've migrated to the new `webpackAlertFilters`.
## Test instructions
- Check out https://code.ogdsoftware.nl/oscar/ggze-mijn-oplossing/tree/offline-werken.
- Check out this jester-branch.
- Create an `npm link` from ggze to your jester-checkout.
- Build ggze and see that no webpack alerts are printed.
- Remove the webpackAlertFilters from ggze, rebuild, and see webpack alert spam.
